macros/utils/table_operations/replace_table_data.sql

```diff
@@ -38,3 +38,15 @@
 
     {% do adapter.drop_relation(intermediate_relation) %}
 {% endmacro %}
+
+{# Glue - truncate and insert (non-atomic) #}
+{% macro glue__replace_table_data(relation, rows) %}
+    {% set intermediate_relation = elementary.create_intermediate_relation(relation, rows, temporary=True) %}
+    {% do dbt.glue_exec_query(dbt.get_insert_overwrite_sql(intermediate_relation, relation)) %}
```

Comment:
Ideally should be using `run_query`, but not able to pass the flag as 'False' for DDL and DML statements which in turn is causing issues at `run_query` when it tries to change the case of columns into lowercase. So using `glue_exec_query` for now.

Note: When DDL and DML statements are passed to `run_query` in dbt it will return `none`